addons/mail/models/mail_mail.py

```diff
@@ -19,6 +19,7 @@
 from odoo.addons.base.models.ir_mail_server import MailDeliveryException
 
 _logger = logging.getLogger(__name__)
+_UNFOLLOW_REGEX = re.compile(r'<p id="mail_unfollow".*?<\/p>', re.DOTALL)
```

Comment:
no need to create a global variable just for that (it's used only once)

Reply:
Actually, I have imported it also in the test to check in test_initial_data that the template and the regexp works together. Also it allows to compile it once for all. What do you think ? (I can also do simpler test or duplicate it in the test)

Reply:
ok for me :)